src/main/cpp/CMakeLists.txt

```
@@ -204,6 +205,7 @@ add_library(
   src/SparkResourceAdaptorJni.cpp
   src/SubStringIndexJni.cpp
   src/ZOrderJni.cpp
+  src/HLLPP.cu
```

How about `HyperLogLogPP`?
```suggestion
  src/HyperLogLogPP.cu
```

---

Done.